Projektskabelon-master/src/main/java/FunctionLayer/OverlayMaterialCalculator.java

```diff
@@ -189,13 +193,15 @@ public  static Material screwForOverlayOneWall ( Wall wall, String overlayName)
             overDoorSpaer.setComment("Horizontal framing-dør");
             overDoorSpaer.setSize(DOORWIDTH + (int) 1.5 * POSTWIDTH);
             overDoorSpaer.setAmount(overDoorSpearQuantity);
+            overDoorSpaer.setPrice(LogicFacade.getPrice(overDoorSpaer.getId()));
```

Comment:
If you set the price here, it will be calculated for the needed size, not for the available one. I calculate de available sizes first in function layer